src/eei.cpp

```diff
@@ -224,6 +224,29 @@ Literal EthereumInterface::callImport(Import *import, LiteralList& arguments) {
       return Literal();
     }
 
+    if (import->base == Name("log")) {
+      cout << "log" << endl;
+
+      uint32_t dataOffset = arguments[0].geti32();
+      uint32_t length = arguments[1].geti32();
+      uint32_t numberOfTopics = arguments[2].geti32();
+
+      uint8_t data[length];
```

Comment:
I think we have to avoid allocating large data on the stack. Can you go for the moment with the messy `vector` allocation (also avoids the byteswapping)?